adapters/base/src/main/java/org/apache/cassandra/sidecar/adapters/base/NodeInfo.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.adapters.base;
+
+/**
+ * Holder class for Node related
+ */
+public class NodeInfo
```

Comment:
I am +0 on this change, there's nothing that we leverage from the enum effectively

Reply:
Like I said earlier, I am in favor of enums (not necessarily in this holder class) for improved readability and maintainability of related constants, which in this case, relates to node states and statuses. 

This is how Cassandra manages these constants as well
https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/service/StorageService.java#L423

@frankgh - Do you see any reasons to not go down this route?